lib/pluginmanager/install.rb

```diff
@@ -123,12 +123,30 @@ def validate_cli_options!
   # Check if the specified gems contains
   # the logstash `metadata`
   def verify_remote!(gems)
+    gems_swap = {}
     options = { :rubygems_source => gemfile.gemset.sources }
     gems.each do |plugin, version|
       puts("Validating #{[plugin, version].compact.join("-")}")
       next if validate_plugin(plugin, version, options)
+
+      signal_usage_error("Installs of an alias doesn't require version specification --version") if version
+
+      # if the plugin is an alias then fallback to the original name
+      if LogStash::PluginManager::ALIASES.has_key?(plugin)
+        resolved_plugin = LogStash::PluginManager::ALIASES[plugin]
```

Comment:
We should bring the usage error down into the block that ensures it only runs when we are dealing with an alias -- otherwise it will also fire when we are attempting to install _any_ plugin that doesn't pass `validate_plugin`.

This also ensures that we have more context and can give better guidance with our error message. I've also adjusted the wording to match the tone of other error messages in this PR.

```suggestion

      # if the plugin is an alias then fallback to the original name
      if LogStash::PluginManager::ALIASES.has_key?(plugin)
        resolved_plugin = LogStash::PluginManager::ALIASES[plugin]
        signal_usage_error("Cannot install a specific version of #{plugin}, which is an alias for #{resolved_plugin}; if you wish to install a specific version of #{resolved_plugin}, you can do so directly.") if version
```